### PR TITLE
CI: Don't trigger runtime pipelines on perf pipeline only changes

### PR DESCRIPTION
**Pull request by radical**

### CODE REVIEW

File: eng/pipelines/common/evaluate-default-paths.yml [hunk 1]
```diff
@@ -52,6 +52,14 @@ parameters:
         eng/testing/bump-chrome-version.proj
         eng/testing/ChromeVersions.props
     ]
+    _perf_pipeline_specific_only: [
+        eng/pipelines/runtime-wasm-perf.yml
+        eng/pipelines/coreclr/perf*yml
```
Comment (cincuranet) [inline eng/pipelines/common/evaluate-default-paths.yml:57]: I guess this should be `perf*.yml`.

File: eng/pipelines/common/evaluate-default-paths.yml [hunk 2]
```diff
@@ -52,6 +52,14 @@ parameters:
...
+        eng/pipelines/coreclr/templates/perf-job.yml
```
Comment (cincuranet) [inline eng/pipelines/common/evaluate-default-paths.yml:58]: This is covered by the previous match.

File: eng/pipelines/coreclr/templates/run-performance-job.yml [hunk 3]
```diff
@@ -26,7 +26,7 @@ parameters:
   javascriptEngine: 'NoJS'
   helixTypeSuffix: ''             # optional -- appends to HelixType
   collectHelixLogsScript: ''      # optional -- script to collect the logs, and artifacts helpful for debugging failures
-  additionalSetupParameters: ''   # optional -- additional setup parameters that are job-specific
+  additionalSetupParameters: ''   # optional -- additional setup parameters that are job-specific -dummy
```
Comment (cincuranet) [inline eng/pipelines/coreclr/templates/run-performance-job.yml:29]: Is this change intentional?

File: eng/pipelines/common/evaluate-default-paths.yml (hunk 2 above)
Comment (cincuranet) [reply eng/pipelines/common/evaluate-default-paths.yml:58]: Ah, sorry. I missed the `templates` in path.

**Review by cincuranet — APPROVED** [review]

**Review by LoopedBard3 — APPROVED** [review]

File: eng/pipelines/coreclr/templates/run-performance-job.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (radical) [reply eng/pipelines/coreclr/templates/run-performance-job.yml]: Done.
